README.md

```diff
@@ -220,6 +223,7 @@ spec:
       worker-0: "6233"
       worker-1: "6234"
       worker-2: "6235"
+  remediationStrategy: OutOfServiceTaint
```

Comment:
Although `OutOfServiceTaint` is preferable when it is enabled, but by default the `remediationStrategy` will be `ResourceDeletion` (and it is by default because due to issue of support to the _OutOfService_ taint)